.github/ISSUE_TEMPLATE/OPEV.mv

```diff
@@ -0,0 +1,121 @@
+---
+name: OpenVEX Enhancement Proposal (OPEV)
+about: Propose a change for the OpenVEX project
+labels: opev
+
+---
+<!-- 
+
+    Follow the template's instructions to open a new Enhancement
+    Proposal.
+-->
+
+# OPEV-00: My Awesome Feature
+
+<!--
+    Please choose a title that describes your enhancement.
+    After submitting the PR, the OPEV number will be the 
+    PR number assigned by GitHub.
+
+    Please edit the PR after submitting and set the correct
+    number and use the same title here in the body and in the
+    PR iteself.
+-->
+
+## 🖊️ Enhancement Overview
```

Comment:
nit: perhaps have a section on projects affected so that the right set of folks can help track, as there are more repos, it will be helpful at a first glance to see what is relevant for different folks

Reply:
Good idea @lumjjb , I've added a section to list the projects. Thanks!